### PR TITLE
DCK-30 Added sponsors paragraph

### PR DESCRIPTION
**Pull request by andriyun**
## Issue

> Added sponsors paragraphs #30 

## Steps for review:

- [ ] Create sponsor basic page for next step
- [ ] Create landing page
- [ ] Add sponsors paragraph  to content (any other) section
- [ ] Fill all fields for Sponsors section/item and save landing page 
- [ ] You should get sponsors render as on screenshot

image

https://app.zeplin.io/project/5c4cbebfc7769b408463817d/screen/5c4cc1ee3f49dc410a23f945

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer.
- [ ] All tests are running and there are no failed tests reported by CI.
- [ ] Documentation has been updated according to PR changes.
- [ ] Steps for review have been provided according to PR changes.

Thank you for your сontribution and help to Drupal Ukraine!

### CODE REVIEW

**Comment by ivan-berezhnov** [comment]
Hi @andriyun 
Could you please fix a couple of items:
1. Please use color fields for colours
screen shot 2019-03-07 at 6 49 31 pm
2. If added only one section this section should have full wight 
screen shot 2019-03-07 at 6 52 51 pm
3. Please add space between images in section
4. Heding should be 45px

**Comment by andriyun** [comment]
@ivan-berezhnov 
1. I've used for color field I've used approach from existing field with color.
See https://github.com/drupal-ukraine/dcamp-site/pull/68/files#diff-8157913b8ef17633299a322b8fb5f819
2. Added logic to use classes for single section and more the one (grid) with proper styles
3. Done
4. Done

@anpolimus Could you help me find out why build are failed.
I didn't found any info why it's failed
Circle CI says
```
ssh: Could not resolve hostname 138.197.180.59.nip.io: Try again
Notifying Github Status API: ci/docksal => failure
Exited with code 255
```

File: docroot/themes/custom/dckyiv/style/scss/components/_paragraph--sponsor-section.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  &--header {
+    display: flex;
+    justify-content: center;
+    font-family: $font-proxima-nova-bold;
+    font-size: 1.8rem;
+    text-align: center;
+    text-transform: uppercase;
+    color: $color-white;
+    padding: 8px 8px 10px;
```
Comment (ivan-berezhnov) [inline docroot/themes/custom/dckyiv/style/scss/components/_paragraph--sponsor-section.scss]: can you please remove padding and add:
     display: flex;
     align-items: center;

File: docroot/themes/custom/dckyiv/style/scss/components/_paragraph--sponsor-section.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    border: 1px solid $color-black;
+    border-top: 0;
+    display: flex;
+    align-items: center;
+    height: calc(100% - 45px);
+  }
+
+  .field--name-field-sponsor-items {
+    display: flex;
+    flex-direction: row;
```
Comment (ivan-berezhnov) [inline docroot/themes/custom/dckyiv/style/scss/components/_paragraph--sponsor-section.scss]: Please add a column for tablet and mobile version
Or do something if small spaces 
Screen Shot 2019-03-12 at 2 19 53 PM

**Comment by anpolimus** [comment]
@andriyun I've fixed builds.

**Comment by ivan-berezhnov** [comment]
@anpolimus My result 
Screen Shot 2019-03-14 at 12 53 18 PM

**Pull request closed without merging**